spyder/app/tour.py

```diff
@@ -50,12 +52,17 @@ class SpyderWidgets(object):
     # Panes
     ipython_console = 'ipyconsole'
     editor = 'editor'
-    editor_line_number_area = 'editor.get_current_editor().linenumberarea'
+    editor_line_number_area = (
+        'editor.get_current_editor().panels._panels.get(1).values()')

```

Comment:
```suggestion
    panel = Panel.Position.LEFT
    editor_line_number_area = (
        f'editor.get_current_editor().panels._panels[{panel}].values()')
```

Get actual panel ID value instead of hardcoding it; use shorter and more conventional dict access (which fails early)